snapcraft/plugins/rust.py

```diff
@@ -152,6 +152,13 @@ def _fetch_rust(self):
                   '--disable-sudo', '--save'] + options)
 
     def _fetch_deps(self):
+        
+        source_subdir = getattr(self.options, 'source_subdir', None)
```

Comment:
it still says that source_subdir is not defined. I think we have to have the statement above it that defines it